buildpipeline/portable-linux.groovy

```diff
@@ -61,11 +61,13 @@ simpleDockerNode('microsoft/dotnet-buildtools-prereqs:rhel7_prereqs_2') {
                                      'Ubuntu.1404.Amd64.Open',
                                      'Ubuntu.1604.Amd64.Open',
                                      'Ubuntu.1610.Amd64.Open',
-                                     // 'Ubuntu.1704.Amd64.Open',
                                      'suse.422.amd64.Open',
```

Comment:
Nop, it's not case-sensitive. For example, even though the real queue name for fedora 2.5 is "Fedora.25.Amd64.Open", it still work with current usage 'fedora.25.amd64.Open'.

The queue names are mostly following right naming convention and in upper cases, except for suse ones. This will be corrected in task:
https://github.com/dotnet/core-eng/issues/908